### ISSUE TITLE
Significant performance drop

### ISSUE THREAD

**Issue by aurelien-reeves** [Shippable/support#4527]
We recently noticed an important drop in execution time of our builds.

To confirm the drop I've rebuild an old build.

Original: https://app.shippable.com/github/hiptest/hiptest/runs/5972/
Now: https://app.shippable.com/github/hiptest/hiptest/runs/6022/

We can see that one the third // job was executed in 10 minutes, now 13 minutes.
More significant, the second one was first executed in 21 minutes, now 32 minutes.

**Comment by rageshkrishna** [comment]
@aurelien-reeves Is it alright if I try re-building the same job once again? I'd like to monitor the build to see what is slowing it down.

**Comment by rageshkrishna** [comment]
@aurelien-reeves I think I understand the problem here. The performance impact began on the 17th of August, when we updated your subscription to use a newer version of our build runner. Unfortunately, that particular release had a CPU usage issue in the build runner which made your build times shoot up to 45+ minutes.

Prior to the 17th of August, I can see that successful builds on this branch were taking somewhere between 27 to 35 minutes.

Yesterday we did another release of the build runner that fixes the CPU usage issue, and I see that today's build ran in 32 minutes, which seems to be back in the same range as it was before. I think you'll find that if you run a few more builds they will all stay in the same ballpark as before.

Please let me know if this isn't the case.

**Comment by aurelien-reeves** [comment]
Hi,

Thank you for your quick response.

If you need triggering re-build of jobs, please do, it is alright.

We will monitor build execution time today, and keep you informed.

Thank you,
Regards,
Aurélien

**Comment by aurelien-reeves** [comment]
It is better now, but it remains significantly slower than it was before

**Comment by aurelien-reeves** [comment]
Nothing new about that?

**Comment by manishas** [comment]
@aurelien-reeves let me chat with the team and get back to you asap after the long weekend.

**Comment by ric03uec** [comment]
@aurelien-reeves we're still debugging the performance issues with the new node types on which your subscription builds were running. Meanwhile, we've moved the subscription back to the previous nodes which should get the build times back to their previous numbers. 
We've identified a potential root cause for this but will need more testing and analysis to confirm.
We'll post an update on this issue once we have more information.
